chains/evm/contracts/test/ccvs/CommitteeRamp/CommitteeRamp.applyAllowlistUpdates.t.sol

```diff
@@ -1,18 +1,18 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.24;
 
-import {BaseOnRamp} from "../../../onRamp/BaseOnRamp.sol";
-import {CommitOnRamp} from "../../../onRamp/CommitOnRamp.sol";
-import {CommitOnRampSetup} from "./CommitOnRampSetup.t.sol";
+import {CommitteeRamp} from "../../../ccvs/CommitteeRamp.sol";
+import {BaseOnRamp} from "../../../ccvs/components/BaseOnRamp.sol";
+import {CommitteeRampSetup} from "./CommitteeRampSetup.t.sol";
 
-contract CommitOnRamp_applyAllowlistUpdates is CommitOnRampSetup {
+contract CommitteeRamp_applyAllowlistUpdates is CommitteeRampSetup {
   function setUp() public override {
     super.setUp();
 
     // Enable allowlist for destination chain once.
     BaseOnRamp.DestChainConfigArgs[] memory destChainConfigs = new BaseOnRamp.DestChainConfigArgs[](1);
     destChainConfigs[0] = _getDestChainConfig(s_router, DEST_CHAIN_SELECTOR, true);
-    s_commitOnRamp.applyDestChainConfigUpdates(destChainConfigs);
+    s_commitRamp.applyDestChainConfigUpdates(destChainConfigs);
```

Comment:
imo it should be committeeCCV not ramp, if that's not in the spec let's discuss with Connor&Makram

Reply:
https://github.com/smartcontractkit/chainlink-ccip/pull/1182/commits/826eb502e9d11aebd9b9ad9a273b6af3813bae3e